demos/react-dapp/src/App.tsx

```diff
@@ -239,6 +273,36 @@ const App: React.FC = () => {
     return { transaction: transactionSigned }
   }
 
+  // Create multi-signature account
+  const handleCreateMultisigAccount = async () => {
+    // Fetch public keys from mirror node for each account
+    const fetchPublicKey = async (accountId: string) => {
+      const response = await fetch(
```

Comment:
I would strongly consider creating a retryable logic mirror node service and I would recommend adding configuration for mirrornode env (testnet/mainnet)

The former would be there to protect against mirrormode being overwhelmed and been out to elegantly handle any state, there isn't any try catch in this function, then a timeout issue will cascade the lib.

Would also recommend enabling different mirrornode providers (arkhia et al) where auth is handled, for prod usecases.

Reply:
> I would strongly consider creating a retryable logic mirror node service and I would recommend adding configuration for mirrornode env (testnet/mainnet)
> 
> The former would be there to protect against mirrormode being overwhelmed and been out to elegantly handle any state, there isn't any try catch in this function, then a timeout issue will cascade the lib.
> 
> Would also recommend enabling different mirrornode providers (arkhia et al) where auth is handled, for prod usecases.

Since this is just the React demo, I don't think it makes a ton of sense setting up something so complex